f2/src/forms/ImpactStatementInfoForm.js

```diff
@@ -81,23 +82,27 @@ export const ImpactStatementInfoForm = props => {
               >
                 <Control as="fieldset" name="howWereYouAffected">
                   <FormLabel as="legend" htmlFor="howWereYouAffected" mb={2}>
-                    <Text fontWeight="bold">
-                      <Trans id="impactPage.detail" />
-                    </Text>
+                    <Trans id="impactPage.detail" />
                   </FormLabel>
                   <Stack spacing={4} shouldWrapChildren>
                     {howWereYouAffected.map(key => {
+                      const checked = getImpact(
+                        client,
+                      ).howWereYouAffected.includes(key)
                       return (
-                        <CheckboxArrayControl
-                          key={key}
-                          name="howWereYouAffected"
-                          value={key}
-                          isChecked={getImpact(
-                            client,
-                          ).howWereYouAffected.includes(key)}
-                        >
-                          {i18n._(key)}
-                        </CheckboxArrayControl>
+                        <Box>
+                          <CheckboxArrayControl
+                            key={key}
+                            name="howWereYouAffected"
+                            value={key}
+                            isChecked={checked}
+                          >
+                            {i18n._(key)}
+                          </CheckboxArrayControl>
+                          <ConditionalForm show={checked}>
+                            Conditional form
+                          </ConditionalForm>
```

Comment:
Was this ConditionalForm for debugging?

Reply:
Yes, I was working on it last night and I could not finish it. Did not realize that it would append to my PR. See the next commit to find working placeholder conditional forms!

We can discuss how we want to implement conditional fields. I was thinking we could place them in another file. I'm a little worried about readability... there must be a better way

Reply:
looks good! only thing I'm seeing is when I load the page the fonts are initially different and then change after a fraction of a second. Any thoughts about what's doing that?

Reply:
I'm loading a google font. Maybe I'm not doing it efficiently. Look for it in app.js
I was looking for a better way to load it, maybe as a promise, but could not find anything. Lets do an issue to fix this

Reply:
I see. Yeah, we can push that to another PR. Probably we can just download the fonts and include them in the repo, then serve them locally.